Whiskey/Functions/Invoke-WhiskeyDotNetCommand.ps1

```diff
@@ -0,0 +1,92 @@
+
+function Invoke-WhiskeyDotNetCommand

```

Comment:
I'm wondering if this should be a `DotNet` task that the other tasks call? It would only have two properties: `Command`, the name of the .NET command to run, and `Argument`, a list of arguments to pass to the command? Or maybe just the `ArgumentList` property?

```
Build:
- DotNet:
    ArgumentList: pack
```

What do you think?

Reply:
I do like that idea, it's clean and adds additional flexibility to the whiskey.yml but at the same time it adds additional overhead of setting up and running a whole other task. But that additional overhead might be negligible in the grand scheme of things.

Though it does add the possibility of running into random edge cases where a task is calling another task like we had when we added the `Parallel` task.

Reply:
Let's just be done. Leave as-is.